zerver/lib/notifications.py

```diff
@@ -279,9 +279,24 @@ def do_send_missedmessage_events_reply_in_zulip(user_profile, missed_messages, m
     senders = set(m.sender.full_name for m in missed_messages)
     sender_str = ", ".join(senders)
     plural_messages = 's' if len(missed_messages) > 1 else ''
+    realm_str = missed_messages[0].get_realm().string_id
+    if (missed_messages[0].recipient.type == Recipient.HUDDLE):
+        disp_recipient = get_display_recipient(missed_messages[0].recipient)
+        other_recipients = [r['full_name'] for r in disp_recipient
+                                if r['email'] != user_profile.email]
+        if len(other_recipients) == 2:
+            subject = u"Group PM%s with %s in %s" % (plural_messages, " and ".join(other_recipients), realm_str)
+        elif len(other_recipients) == 3:
+            subject = u"Group PM%s with %s, %s, and %s in %s" % (plural_messages, other_recipients[0], other_recipients[1], 
```

Comment:
Here. The [error message](https://travis-ci.org/zulip/zulip/jobs/228585987) from Travis says there's a trailing whitespace at the end of this file.